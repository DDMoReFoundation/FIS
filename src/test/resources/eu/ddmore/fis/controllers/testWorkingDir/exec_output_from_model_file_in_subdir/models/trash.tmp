UseCase1.lst
newline
OUTPUT
        1 file(s) copied.
