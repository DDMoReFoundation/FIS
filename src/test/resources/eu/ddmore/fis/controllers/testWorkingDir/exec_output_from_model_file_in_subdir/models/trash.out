Could Not Find C:\Temp\mifshare\ba6a9bad-b13e-4d20-b3c4-d10b9a02519c\models\PARALLEL.pnm
